web3/_utils/module_testing/eth_module.py

```diff
@@ -1182,6 +1173,13 @@ async def test_eth_create_access_list(
         assert len(access_list[0]["storageKeys"][0]) == 32
         assert int(response["gasUsed"]) >= 0
 
+        # assert the result can be used directly in a transaction dict
+        txn["accessList"] = response["accessList"]
```

Comment:
Well, if this works out of the box, then you probably can revert some of your fixes in the documentation.
Specifically, where the documentation describes this is part of the output of that function:
```
            'accessList': [
                AttributeDict({
                    'address': '0xde0B295669a9FD93d5F28D9Ec85E40f4cb697BAe',
                    'storageKeys': [
                        HexBytes('0x0000000000000000000000000000000000000000000000000000000000000003'),
                        HexBytes('0x0000000000000000000000000000000000000000000000000000000000000007'),
                    ]
                }),
                AttributeDict({
                    'address': '0xBB9bc244D798123fDe783fCc1C72d3Bb8C189413',
                    'storageKeys': []
                }),
            ],
```
If the `AttributeDict` and `HexBytes` are really part of this output, then I'd expect `txn` to become illegal (cannot be unmarshalled by Go, or something similar to that).

Reply:
We should be handling that in our formatters before it gets passed to the client